jupyterlab_gallery/manager.py

```diff
@@ -69,6 +69,12 @@ def __init__(self, *args, **kwargs):
         config=True,
     )
 
+    hideGalleryWithoutExhibits = Bool(
```

Comment:
Sorry I have not caught it earlier - since all traitlets use `snake_case` (https://jupyter-server.readthedocs.io/en/latest/other/full-config.html#other-full-config) this should probably be snake case too:

```suggestion
    hide_gallery_without_exhibits = Bool(
```